types/src/discovery_set.rs

```diff
@@ -86,6 +94,30 @@ impl DiscoverySet {
     pub fn from_bytes(bytes: &[u8]) -> Result<Self> {
         lcs::from_bytes(bytes).map_err(Into::into)
     }
+
+    #[cfg(any(test, feature = "fuzzing"))]
```

Comment:
considering all the cfg(any(test) craziness, it might be better to make a module and then re-export it.

Reply:
ahh yeah, I was on the fence about this. the imports look especially ugly with all of the `#[cfg(..)]` noise going on :(. that's a good suggestion, I'll move this into a separate module.